extension/intents/nicknames/nicknames.toml

```diff
@@ -60,4 +60,22 @@ match = """
 phrase = "delete the page called news"
 
 [[nicknames.removePageName.example]]
-phrase = "remove this tab name"
\ No newline at end of file
+phrase = "remove this tab name"
+
+[nicknames.pause]
+description = "Pauses a routine"
+match = """
+  pause routine
```

Comment:
For a message this could also have a phrase like `pause routine (with |) (and |) (show |) (message | status | text | print) [message]`